versions.cfg

```diff
@@ -43,4 +43,11 @@ repoze.who = 2.2
 RestrictedPython = 3.6.0
 zope.sequencesort = 4.0.1
 zope.testbrowser = 3.11.1
+ZODB = 4.2.0
+sphinx-rtd-theme = 0.1.9
+alabaster = 0.7.7
+Babel = 2.2.0
+snowballstemmer = 1.2.1
+
+# only for buildout dependencies
 ZODB3 = 3.11.0
```

Comment:
I imagine changing the ZODB3 dependency in these packages to ZODB is hairy, because of Zope 2.x depending on ZODB3.

+1 for removing the ZODB3 dependency if above is true.